packages/lesswrong/server/vulcan-lib/apollo-server/initGraphQL.ts

```diff
@@ -58,7 +59,7 @@ import { multiDocumentMutations, multiDocumentTypeDefs } from '@/server/resolver
 import { spotlightGqlMutations, spotlightGqlTypeDefs } from '@/server/resolvers/spotlightResolvers';
 import { typingIndicatorsGqlMutations, typingIndicatorsGqlTypeDefs } from '@/server/resolvers/typingIndicatorsResolvers';
 import { acceptCoauthorRequestMutations, acceptCoauthorRequestTypeDefs } from '@/server/acceptCoauthorRequest';
-import { bookmarkGqlMutations, bookmarkGqlTypeDefs } from '@/server/bookmarkMutation';
+import { bookmarkGqlTypeDefs, bookmarkGqlMutations } from '@/server/collections/bookmarks/mutations';
```

Comment:
this import should ideally go into the same section as every other set of collection-level mutation imports.